lisa/sut_orchestrator/azure/common.py

```diff
@@ -110,6 +111,19 @@ class AzureNodeSchema:
         default=None, metadata=field_metadata(data_key="shared_gallery")
     )
     vhd: str = ""
+    security_profile: Optional[SecurityProfileType] = field(
```

Comment:
Not add it into the node space. It's a general feature, doesn't like disk or network.

Reply:
Should it be in AzurePlatformSchema?
I'm having a hard time figuring out how to get it from AzurePlatformSchema to settings or arm_parameters

Reply:
It should be added automatically into features of a capability.

Reply:
I don't understand.

1. Which schema should the security_profile be in
2. Where should security_profile be in the runbook
3. How does the field security_profile get turned into a SecurityProfileSettings

Reply:
1. The schema itself is in right place already (features/security_profile)
2. In runbook, it's a settings schema. Schema is Settings, the SecurityProfile is the feature class, not schema. I have a try, it  should be added like below, but it doesn't work. I will fix it.
3. It's not valid, the settings will convert to a feature in Features class.

![image](https://user-images.githubusercontent.com/27178119/185508757-dae7c43c-2be8-4dac-9a72-b84cf0463c15.png)

Reply:
PS: it should work as I verified. Let me know if there is any problem.